check_log_ng.py

```diff
@@ -1272,6 +1283,13 @@ def _make_parser():
         default=False,
         help=("QUIET mode: Suppress the output of matched lines.")
     )
+    parser.add_argument(
+        "--error-no-exist",
+        action="store_true",
+        dest="error_no_exist",
+        default=False,
+        help=("If file not exist, return UNKNOWN error.")
```

Comment:
I think we should use one of "not found" or "not exist" in all messages.

Reply:
OK. Use only "not exist".